### DISCUSSION TITLE
How to change worker_processes?

### DISCUSSION THREAD

**Discussion by HideCM** [NginxProxyManager/nginx-proxy-manager#3909] [Q&A]
`2024/07/31 11:40:27 [alert] 191#191: *102555 512 worker_connections are not enough while connecting to upstream, client: 172.71.10.1, server: domain1.com, request: "GET /phim/13-gio-linh-ngam-benghazi HTTP/2.0", upstream: "http://194.233.1.1:81/phim/13-gio-linh-ngam-benghazi", host: "domain1.com"`

Anyone can help me? I installed by docker, and I can't find the **nginx.conf** on my VPS

**Comment by rodrigolyamamoto** [comment]
Create a file named `events.conf` with desired configuration (like `worker_connections  10240;`) and put in `data/nginx/custom/` folder.
